pymbolic/mapper/unifier.py

```diff
@@ -43,9 +43,8 @@ def unify_map(map1, map2):
 
 
 class UnificationRecord(object):
-    def __init__(self, equations, lmap=None, rmap=None):
-        self.equations = equations
 
+    def __init__(self, equations, lmap=None, rmap=None):
         # lmap and rmap just serve as a tool to reject
```

Comment:
That was not the intention. I found another way to get the behavior that I wanted (0be9387).